api/v1beta1/openstackdataplaneservice_types.go

```diff
@@ -70,7 +70,7 @@ type OpenStackDataPlaneServiceSpec struct {
 
 	// OpenStackAnsibleEERunnerImage image to use as the ansibleEE runner image
 	// +kubebuilder:validation:Optional
-	OpenStackAnsibleEERunnerImage string `json:"openStackAnsibleEERunnerImage,omitempty" yaml:"openStackAnsibleEERunnerImage,omitempty"`
```

Comment:
This yaml tag is required b/c gopkg.in/yaml.v3 marshals/unmarshals struct fields as all lowercase only. So, we need to tell it to use the k8s camelCase convention instead.